app/presenters/tree_builder.rb

```diff
@@ -113,7 +113,7 @@ def self.root_options(tree_name)
       ["CFME Region: #{region.description} [#{region.region}]", "CFME Region: #{region.description} [#{region.region}]", "miq_region"]
     when :dialogs_tree                  then [_("All Dialogs"),                  _("All Dialogs")]
     when :dialog_import_export_tree     then [_("Service Dialog Import/Export"), _("Service Dialog Import/Export")]
-    when :export_tree                   then [_("Import / Export"),              _("Import / Export"), _("report")]
+    when :export_tree                   then [_("Import / Export"),              _("Import / Export"), "report"]
```

Comment:
Perhaps changing to symbol or constant will make it a bit more obvious that this is not *just* a string.